fortuna/src/login/Login.js

```diff
@@ -3,37 +3,120 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import LoginPopup from './LoginPopup.js';
+//import LoginPopup from './LoginPopup.js';
 import './Login.css';
-
+const PORT = 3000;
 // Login component.
-class Login extends React.Component<{||}> {
+type Props = {||}; 
+type State = {|
+    response: string,
+    userName: string,
+    password: string,
+    responseToPost: string,
+  |};
```

Comment:
nit: make the closing |} indentation match the indentation level of type (like you would with a for loop or method).

Thanks for fixing this though :)